index.bs

```diff
@@ -640,20 +644,26 @@ dictionary SelectSpatNavBestCandidateOptions {
 };
 
 partial interface Element {
-    sequence&lt;Node> findSpatNavCandidates(FindSpatNavCandidatesOptions arg);
+    Node getSpatnavContainer();
```

Comment:
I understand this is only a convenience method... but.... getting the "spatnav container" can be done with vanilla JS...

Isn't it better to leave polyfillable things out from the spec? At least until JS framework ask for it being done natively.

In principle, I'd like the spec to focus on adding/enabling new use case that's not already possible in plain JS... :) 

If you do have to add this method, can't it be called something that hints for use cases that aren't related to spatnav. Maybe "enclosingScoller()"?

Reply:
In Blink we call it "scrollableAreaOrDocumentOf()"... How does that sound?

Reply:
As you say, it can be done in JS. But it is a nice convenience. How about meeting in the middle, and keeping it for now, but marking it "at risk" to signal to implementors that we don't consider this a MUST HAVE, and are ok with dropping it if they don't want to implement.

As for the name, I'd rather not use `enclosingScroller()` or `scrollableAreaOrDocumentOf()`, because these names would become misleading when the `spatnav-container` property, or any other mechanism to turn non scrollers into spatnav containers.

Reply:
I thought we were removing the `spatnav-container` property.

OK for marking it "at risk".